cmd/kube-scheduler/app/options/options.go

```diff
@@ -150,13 +150,33 @@ func newDefaultComponentConfig() (*kubeschedulerconfig.KubeSchedulerConfiguratio
 // Flags returns flags for a specific scheduler by section name
 func (o *Options) Flags() (nfs cliflag.NamedFlagSets) {
 	fs := nfs.FlagSet("misc")
-	fs.StringVar(&o.ConfigFile, "config", o.ConfigFile, `The path to the configuration file. The following flags can overwrite fields in this file:
+	fs.StringVar(&o.ConfigFile, "config", o.ConfigFile, `The path to the configuration file. The affected parameters are as follows.
+The following flags can overwrite fields in this file:
+  --algorithm-provider
+  --policy-config-file
```

Comment:
IRRC policy flags cause an error when used with config

Reply:
I enabled policy-config to be set with componentconfig (to ease the deprecation transition) in https://github.com/kubernetes/kubernetes/pull/92531